components/bio-formats-plugins/src/loci/plugins/util/ROIHandler.java

```diff
@@ -311,19 +337,48 @@ else if(ijRoi.getTypeAsString().matches("Rectangle")){//Check if its a Rectangle
 
     // -- Helper methods --
 
+    private static void storeText(TextRoi roi, MetadataStore store, int roiNum, int shape) {
+
+        store.setLabelX(roi.getPolygon().getBounds().getX(), roiNum, shape);
+        store.setLabelY(roi.getPolygon().getBounds().getY(), roiNum, shape);
+
+        store.setLabelText(roi.getText().trim(), roiNum, shape);
+        if (roi.getStrokeWidth() > 0) {
+            store.setLabelStrokeWidth( new Length((roi.getStrokeWidth()), UNITS.PIXEL), roiNum, shape);
+        }
+        if (roi.getStrokeColor() != null) {
+            store.setLabelStrokeColor(new ome.xml.model.primitives.Color(roi.getStrokeColor().getColorSpace().getType()) , roiNum, shape);
```

Comment:
No, there is an ome.xml.model.primitives.Color constructor that accepts the red, green, blue and alpha values individually and performs the conversion internally.

See http://downloads.openmicroscopy.org/bio-formats/5.0.6/api/ome/xml/model/primitives/Color.html#Color(int,%20int,%20int,%20int)